cmsranking/static/Ranking.css

```diff
@@ -922,6 +922,7 @@ abbr {
     right: 470px;
     font-size: 2em;
     line-height: 1.5em;
+    width: 100px;
```

Comment:
This turns out to be buggy because it also restricts the width of `f_name`, `l_name`, `team` etc :') (this CSS affects 4 classes)

Will not include this in the final branch.